src/index.js

```diff
@@ -242,17 +243,24 @@ class RuntimeMetrics {
   }
 
   start() {
+    if (this.started) {
+      this.registry.logger.info('nflx-spectator-nodejsmetrics already started');
```

Comment:
Were we seeing multiple starts in practice?

Reply:
No. The request was just to provide the `started` property.